openehr-rm/src/main/java/com/nedap/archie/rm/ehr/EhrStatus.java

```diff
@@ -22,9 +23,11 @@
     private PartySelf subject;
     @XmlElement(name="is_queryable")
     @RMProperty("is_queryable")
+    @JsonProperty(value = "is_queryable")
     private boolean isQueryable;
     @XmlElement(name="is_modifiable")
     @RMProperty("is_modifiable")
+    @JsonProperty(value = "is_modifiable")
```

Comment:
The json naming strategy should already add the underscore. Does it automatically strip the is_?

Reply:
In the Branch we used 
  objectMapper.setPropertyNamingStrategy(PropertyNamingStrategy.SNAKE_CASE);

was not set in JacksonUtil

Reply:
But it is, and has been for three years:

https://github.com/openEHR/archie/blob/master/archie-utils/src/main/java/com/nedap/archie/json/JacksonUtil.java#L61

Reply:
Yes saw it. Strange. I will take a look into it tomorrow. Also the other thinks you mentioned.

Reply:
EhrStatusTest.testBooleanSerialization

fails without the Annotations since jackson strips is from boolean values